### PR TITLE
Test: Fix assert message on kafka tests

### PR DESCRIPTION
**Pull request by eloycoto**
Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

### CODE REVIEW

**Comment by eloycoto** [comment]
test-me-please

File: test/runtime/kafka.go [hunk 1]
```diff
@@ -129,9 +129,11 @@ var _ = Describe("RuntimeValidatedKafka", func() {
 		Expect(err).Should(BeNil())
 
 		endPoints, err := vm.PolicyEndpointsSummary()
-		Expect(err).Should(BeNil())
-		Expect(endPoints[helpers.Enabled]).To(Equal(1))
-		Expect(endPoints[helpers.Disabled]).To(Equal(2))
+		Expect(err).Should(BeNil(), "Cannot get policy enforcement summary")
```
Comment (tgraf) [inline test/runtime/kafka.go]: I'm not sure what that means. What command does it run?

File: test/runtime/kafka.go [hunk 2]
```diff
@@ -129,9 +129,11 @@ var _ = Describe("RuntimeValidatedKafka", func() {
...
+		Expect(endPoints[helpers.Enabled]).To(Equal(1),
+			"Kafka container does not have any policy enforcement, and it should")
```
Comment (tgraf) [inline test/runtime/kafka.go]: `"Check number of endpoints with policy enforcement enabled"`

The kafka context is clear from the test context.

File: test/runtime/kafka.go [hunk 3]
```diff
@@ -129,9 +129,11 @@ var _ = Describe("RuntimeValidatedKafka", func() {
...
+		Expect(endPoints[helpers.Disabled]).To(Equal(2),
+			"Number of endpoints without Policy Enforcement mismatch")
```
Comment (tgraf) [inline test/runtime/kafka.go]: `"Check number of endpoints with policy enforcement enabled"`

**Review by tgraf — CHANGES_REQUESTED** [review]

**Comment by eloycoto** [comment]
test-me-please

File: test/runtime/kafka.go [hunk 4]
```diff
@@ -129,10 +129,11 @@ var _ = Describe("RuntimeValidatedKafka", func() {
...
 		endPoints, err := vm.PolicyEndpointsSummary()
-		Expect(err).Should(BeNil())
-		Expect(endPoints[helpers.Enabled]).To(Equal(1))
-		Expect(endPoints[helpers.Disabled]).To(Equal(2))
-
+		Expect(err).Should(BeNil(), "Cannot get endpoint list")
+		Expect(endPoints[helpers.Enabled]).To(Equal(1),
+			"Check number of endpoints with policy enforcement enabled")
+		Expect(endPoints[helpers.Disabled]).To(Equal(2),
+			"Check number of endpoints without policy enforcement")
```
Comment (ianvernon) [inline test/runtime/kafka.go]: without policy enforcement enabled

Comment (joestringer) [reply test/runtime/kafka.go]: perhaps, 'with policy enforcement disabled'? reading double-negatives is always more difficult.

**Comment by manalibhutiyani** [comment]
Looks fine. Please change the wording as per the review comments.

**Review by manalibhutiyani — APPROVED** [review]

**Comment by eloycoto** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Pull request merged**